File: C++ programming/PROBLEMS  ON LOOPING/Program8.cpp
#include<iostream>
using namespace std;

void Display(int iNo)
{
    int iCnt  = 0 ;
    for(iCnt = 1 ; iCnt <= iNo ; iCnt++)
    {
        cout<<"jay Ganesh..."<<endl;
    }
}
int main()
{
    int iValue = 0;
    cout<<"Enter number of time you want Display:"<<endl;
    cin>>iValue;
    Display(iValue);
    return 0 ;
}

File: JAVA Programming/PROBLEM ON LOOPING/Program8.java
import java.util.Scanner;
class Program8
{
    public static void Display(int iNo)
    {
        int iCnt  = 0;
        for(iCnt  = 1; iCnt <= iNo; iCnt++)
        {
            System.out.println("Jay Ganesh...");
        }
    }
    public static void main(String Args[])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the  Number :");
        int iValue = sobj.nextInt();

        Display(iValue);
    }
}